torch/_inductor/decomposition.py

```diff
@@ -282,10 +310,13 @@ def mm(self, input2):
 # - Normalize cat calls, so that legacy empty 1-D tensors are removed (NB: we
 #   don't remove ALL empty tensors, only the naughty ones)
 @register_decomposition([aten.cat.default])
-def cat(tensors, dim=0):
+def cat(
+    tensors: torch.Tensor,
```

Comment:
this should be list of tensors right ?

Reply:
Yep, great catch!

Reply:
Wondering how did mypy not complain...

Reply:
We probably dont invoke this function directly much.. only through decomps